indy_node/server/domain_req_handler.py

```diff
@@ -269,6 +270,12 @@ def _validate_revoc_reg_def(self, req: Request):
         assert revoc_def_tag
         assert revoc_def_type
         tags = cred_def_id.split(":")
+
```

Comment:
Also please add auth checks for RevocRegDefEntry. Make sure that only the owner of RevocRegDef can edit RevocRegEntry